### PR TITLE
Maintenance w/ Recommended Changes

### PR DESCRIPTION
**Pull request by ghost**
made changes to the branch based on the requests in the previous PR

### CODE REVIEW

File: src/namings.rs [hunk 1]
```diff
@@ -80,7 +80,7 @@ pub fn number_to_greek(n: u8) -> String {
                 + "conta"
         }
     } else {
-        panic!(n.to_string().to_owned() + " uncalculatable");
+        panic!("{} uncalculatable", n.to_string().to_owned());
```
Comment (Cxarli) [inline src/namings.rs]: `.to_owned()` should not be necessary here

**Review by Cxarli — CHANGES_REQUESTED** [review]
Just one minor thing, the rest seems fine :blush:

File: src/namings.rs (hunk 1 above)
Comment (ghost) [reply src/namings.rs]: replaced the .to_owned with String::from() in latest commit.

**Review by ghost — COMMENTED** [review]
checked to make sure all looks good, added commits with recommended changes from last PR

**Review by Cxarli — APPROVED** [review]

**Comment by Cxarli** [comment]
It's looking sweet. Thanks again!

**Comment by ghost** [comment]
ahh yeaah. no worries, it was actually fun!

**Pull request merged**